content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -14,9 +14,16 @@ can specify on different Kubernetes components.
 ## Overview
 
 Feature gates are a set of key=value pairs that describe alpha or experimental
-features.
+features.  
 An administrator can use the `--feature-gates` command line flag on each component
-to turn a feature on or off.
+to turn a feature on or off. Each component supports a set of feature gates unique to that component.
+Use `-h` flag to check the list of feature gates for a component.  
```

Comment:
IIRC, `-h` flag still shows a full set of feature gates for all components. It can't show feature gates unique to each component at the moment. I remember @mtaufen has some context on this.

The list from `-h` flag is like below:
```
                APIListChunking=true|false (BETA - default=true)
                APIResponseCompression=true|false (ALPHA - default=false)
                AllAlpha=true|false (ALPHA - default=false)
                AppArmor=true|false (BETA - default=true)
                AttachVolumeLimit=true|false (BETA - default=true)
                BalanceAttachedNodeVolumes=true|false (ALPHA - default=false)
                BlockVolume=true|false (BETA - default=true)
                BoundServiceAccountTokenVolume=true|false (ALPHA - default=false)
                CPUManager=true|false (BETA - default=true)
                CRIContainerLogRotation=true|false (BETA - default=true)
                CSIBlockVolume=true|false (ALPHA - default=false)
                CSIDriverRegistry=true|false (ALPHA - default=false)
                CSINodeInfo=true|false (ALPHA - default=false)
                CustomCPUCFSQuotaPeriod=true|false (ALPHA - default=false)
                CustomPodDNS=true|false (BETA - default=true)
                CustomResourceSubresources=true|false (BETA - default=true)
                CustomResourceValidation=true|false (BETA - default=true)
                CustomResourceWebhookConversion=true|false (ALPHA - default=false)
                DebugContainers=true|false (ALPHA - default=false)
                DevicePlugins=true|false (BETA - default=true)
                DryRun=true|false (BETA - default=true)
                DynamicAuditing=true|false (ALPHA - default=false)
                DynamicKubeletConfig=true|false (BETA - default=true)
                EnableEquivalenceClassCache=true|false (ALPHA - default=false)
                ExpandInUsePersistentVolumes=true|false (ALPHA - default=false)
                ExpandPersistentVolumes=true|false (BETA - default=true)
                ExperimentalCriticalPodAnnotation=true|false (ALPHA - default=false)
                ExperimentalHostUserNamespaceDefaulting=true|false (BETA - default=false)
                HugePages=true|false (BETA - default=true)
                HyperVContainer=true|false (ALPHA - default=false)
                Initializers=true|false (ALPHA - default=false)
                KubeletPodResources=true|false (ALPHA - default=false)
                LocalStorageCapacityIsolation=true|false (BETA - default=true)
                MountContainers=true|false (ALPHA - default=false)
                NodeLease=true|false (ALPHA - default=false)
                PersistentLocalVolumes=true|false (BETA - default=true)
                PodPriority=true|false (BETA - default=true)
                PodReadinessGates=true|false (BETA - default=true)
                PodShareProcessNamespace=true|false (BETA - default=true)
                ProcMountType=true|false (ALPHA - default=false)
                QOSReserved=true|false (ALPHA - default=false)
                ResourceLimitsPriorityFunction=true|false (ALPHA - default=false)
                ResourceQuotaScopeSelectors=true|false (BETA - default=true)
                RotateKubeletClientCertificate=true|false (BETA - default=true)
                RotateKubeletServerCertificate=true|false (BETA - default=true)
                RunAsGroup=true|false (ALPHA - default=false)
                RuntimeClass=true|false (ALPHA - default=false)
                SCTPSupport=true|false (ALPHA - default=false)
                ScheduleDaemonSetPods=true|false (BETA - default=true)
                ServiceNodeExclusion=true|false (ALPHA - default=false)
                StreamingProxyRedirects=true|false (BETA - default=true)
                SupportPodPidsLimit=true|false (ALPHA - default=false)
                Sysctls=true|false (BETA - default=true)
                TTLAfterFinished=true|false (ALPHA - default=false)
                TaintBasedEvictions=true|false (BETA - default=true)
                TaintNodesByCondition=true|false (BETA - default=true)
                TokenRequest=true|false (BETA - default=true)
                TokenRequestProjection=true|false (BETA - default=true)
                ValidateProxyRedirects=true|false (ALPHA - default=false)
                VolumeSnapshotDataSource=true|false (ALPHA - default=false)
                VolumeSubpathEnvExpansion=true|false (ALPHA - default=false)
```

Reply:
Modified.